vimeo-networking/src/main/java/com/vimeo/networking/model/Privacy.java

```diff
@@ -42,35 +42,110 @@
     private static final String PRIVACY_NOBODY = "nobody";
     private static final String PRIVACY_USERS = "users";
     private static final String PRIVACY_ANYBODY = "anybody";
+    private static final String PRIVACY_PRIVATE = "private";
+    private static final String PRIVACY_PUBLIC = "public";
     private static final String PRIVACY_TVOD = "ptv";
     private static final String PRIVACY_CONTACTS = "contacts";
     private static final String PRIVACY_PASSWORD = "password";
     private static final String PRIVACY_DISABLE = "disable";
     private static final String PRIVACY_UNLISTED = "unlisted";
+    private static final String PRIVACY_WHITE_LIST = "white_list";
 
     @UseStag
-    public enum PrivacyValue {
+    public enum PrivacyCommentValue {
```

Comment:
I can't remember if this has been discussed before, but do you think it would be a good idea to remove the `Privacy` prefix from all the names of the enums in this class? Referencing each of these enums seems a little redundant with `Privacy.PrivacyCommentValue`, when `Privacy.CommentValue` seems just as clear. The only downside is that you would always want to use the qualified name for the context.